core/integration/steps/the_markets.go

```diff
@@ -516,9 +534,39 @@ func (r marketRow) lpPriceRange() float64 {
 	return r.row.MustF64("lp price range")
 }
 
+func (r marketRow) linearSlippageFactor() float64 {
+	if !r.row.HasColumn("linear slippage factor") {
+		// set to 1e6 by default
+		return 1000000
```

Comment:
I wouldn't set it this way to 10^6 if the "real" default is 0.1 I would rather add a line to every feature test this impacts to set it "large" there. 

Otherwise, it may be confusing for ppl who don't look here and just look at the default value and then can't figure out maintenance margin numbers.

Reply:
I don't think this should hold off this PR or even be done as part of it even if I agree it makes sense - in case we want to tag and push it to testnet. This change though mechanic will include 10s of FT files.

Reply:
@davidsiska-vega this would add 152 FTs to this PR - I don't mind, but I don't think it's a good idea.

Reply:
Ok I agree, let's do that separately.